src/main/java/duke/manager/input/Parser.java

```diff
@@ -0,0 +1,43 @@
+package duke.manager.input;
+
+public class Parser {
+
+    private String[] arguments;
+    private String input;
+    private boolean isMoreThanTwoWords;
```

Comment:
I like how your use the `is` prefix to make your boolean attribute sound like valid booleans.